grapple/types/pages.py

```diff
@@ -169,6 +169,66 @@ class Meta:
         interfaces = (PageInterface,)
 
 
+def get_preview_page(token):
+    """
+    Get a preview page from a token.
+    """
+    token_params_list = token.split(":").pop(0)
+    token_params_kvstr = token_params_list.split(";")
+
+    params = {}
+    for arg in token_params_kvstr:
+        key, value = arg.split("=")
+        params[key] = value
+
+    """
+    Is there a more convenient way to get at the get_page_from_preview_token
+    method? 
+    """
```

Comment:
Not unless we extract https://github.com/torchbox/wagtail-headless-preview/blob/main/src/wagtail_headless_preview/models.py#L61-L72 into its own method

Reply:
So this is really just a check to see if headless preview is installed and enabled?